backend/tests/test_e2e_markdown_response.py

```
@@ -0,0 +1,233 @@
+"""
+End-to-end test for markdown response functionality.
+"""
+
+import pytest
+import sys
+import os
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)